### PR TITLE
Added F13-24 to the worst spot and is untested

### PR DESCRIPTION
**Pull request by Rionlion100**
Added F13-24 to the end of KB settings and am unable to test it
honestly this was a terrible idea

### CODE REVIEW

**Comment by yanfali** [comment]
please test it.

File: configurator/src/store/modules/keycodes/kb-settings.js [hunk 1]
```diff
@@ -230,5 +230,17 @@ export default [
...
+  { name: 'F15', code: 'KC_F15', title: 'KC_F15' },
+  { name: 'F16', code: 'KC_F16', title: 'KC_F16' },
+  { name: '17F', code: 'KC_F17', title: 'KC_F17' },
+  { name: 'F18', code: 'KC_F18', title: 'KC_F18' },
+  { name: 'F19', code: 'KC_F19', title: 'KC_F19' },
+  { name: 'F20', code: 'KC_F20', title: 'KC_F20' },
+  { name: 'F21', code: 'KC_F21', title: 'KC_F21' },
+  { name: 'F22', code: 'KC_F22', title: 'KC_F22' },
+  { name: 'F23', code: 'KC_F23', title: 'KC_F23' },
+  { name: 'F24', code: 'KC_F24', title: 'KC_F24' },
```
Comment (yanfali) [inline configurator/src/store/modules/keycodes/kb-settings.js]: extra trailing comma

File: configurator/src/store/modules/keycodes/kb-settings.js [hunk 2]
```diff
@@ -230,5 +230,17 @@ export default [
     name: 'RGB Mode G',
     code: 'RGB_M_G',
     title: 'RGB_M_G - RGB Mode: Gradient'
-  }
+  },
+  { name: 'F13', code: 'KC_F13', title: 'KC_F13' },
```
Comment (yanfali) [inline configurator/src/store/modules/keycodes/kb-settings.js]: let's drop the title key. I'm going to make a change to fall back to code if title isn't specified. This is a change I've been meaning to make. Thanks

**Comment by Rionlion100** [comment]
Tested works great in a better place now

File: configurator/src/store/modules/keycodes/kb-settings.js (hunk 2 above)
Comment (Rionlion100) [reply configurator/src/store/modules/keycodes/kb-settings.js]: fixed

**Pull request merged**